src/view/components/AddEntry/index.tsx

```diff
@@ -0,0 +1,192 @@
+    this.state = {
+      volume: '0',
+      quality: '0',
+      costPerUnit: '0',
+    };
+  }
+
+  /** Get current datetime in specified format */
+  private getDatetime = (format: string) => moment().format(format);
+
+  /** Create page buttons */
+  private renderCancelButton = () => this.renderButton('Cancel', 'INFO', this.onCancelPress);
+  private renderSaveButton = () => this.renderButton('Save', 'PRIMARY', this.onSavePress);
+
+  /** Handle pressing cancel button */
+  private onCancelPress = () => this.props.goBack();
+  
+  /** Handle pressing save button */
+  private onSavePress = () => {
+    const timeNow = moment().utc().toString();
+    let newEntry: MilkEntry = {
+      uuid: 'fakeEntryUuid',
+      toUUID: 'fakeToUuid',
+      fromUUID: 'fakeFromUuid',
+      datetime: timeNow,
+      volume: this.state.volume,
+      quality: this.state.quality,
+      costPerUnit: this.state.costPerUnit,
+      lastModified: timeNow,
+      local: 'true',
+    };
+    this.props.createDairy(newEntry);
+    this.props.navigate(Route.MILK_ENTRY_DETAILS);
+  }
+
+  /**
+   * Returns a button with text specified
+   */
+  private renderButton(text: string, color: ButtonColor, onPress: any) {
+    const isInfo = color === 'INFO';
+    const isPrimary = color === 'PRIMARY';
+
+    return (
+      <Col style={Styles.button}>
+        <Button block info={isInfo} primary={isPrimary} onPress={onPress}>
+          <Text>{text}</Text>
+        </Button>
+      </Col>
+    );
+  }
+
+  private renderHeader() {
+    return (
+      <Grid>
+        <Row style={Styles.headerRow}>
+          <H1>
+            {this.props.farmer.firstName} {this.props.farmer.lastName}
+          </H1>
+        </Row>
+        <Row style={Styles.headerRow}>
+          <Text style={Styles.header}>
+            {this.getDatetime('dddd, MMMM DD, YYYY')}
+          </Text>
+        </Row>
+        <Row style={Styles.headerRow}>
+          <Text style={Styles.header}>
+            {this.getDatetime('kk:mm')}
+          </Text>
+        </Row>
+      </Grid>
+    );
+  }
+
+  private renderFields() {
+    return (
+      <Form>
+      <Item floatingLabel>
+        <Label>Amount (L)</Label>
+        <Input />
+      </Item>
+      <Item floatingLabel>
+        <Label>Quality</Label>
+        <Input />
+      </Item>
+      <Item floatingLabel last>
+        <Label>Rate (UGX)</Label>
+        <Input />
+      </Item>
+    </Form>
+    );
+  }
+
+  /**
+   * Render method for AddEntry
+   */
+  public render() {
+    return(
+      <Content padder style={Styles.content}>
+        <List>
+          <ListItem>
+            {this.renderHeader()}
+          </ListItem>
+        </List>
+        {this.renderFields()}
+        <Grid>
+          <Row style={Styles.buttonRow}>
+            {this.renderCancelButton()}
+            {this.renderSaveButton()}
+          </Row>
+        </Grid>
+      </Content>
+    );
+  }
+}
+
+const AddEntryPage = new Composer<NestedPropsType>(AddEntry).page;
```

Comment:
If we aren't creating a fab page we shouldn't inject FabProps

Reply:
You're right, we're not doing fabs on these pages anymore. Fixed.